.github/workflows/guacd-image-scan.yaml

```diff
@@ -0,0 +1,20 @@
+name: Guacd Snyk scan
+
+on:
+  push:
+    paths:
+      - 'guacd/**'
+      - '.github/workflows/**'
+
+jobs:
+  snyk-scan:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run Snyk to check Guacd image for vulnerabilities
+        uses: snyk/actions/docker@master
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+        with:
+          image: guacamole/guacd
+          args: --file=guacd/Dockerfile --severity-threshold=high --exclude-base-image-vulns
```

Comment:
doesn't our previous approach (in CircleCI) have two steps/jobs? One to list the full report and another to block on any `high` ones

Reply:
True, I will add the second one and not make it a pre-req for the PR